contracts/protocol/libraries/logic/ValidationLogic.sol

```diff
@@ -472,6 +472,7 @@ library ValidationLogic {
         .configuration;
       require(!configuration.getPaused(), Errors.RESERVE_PAUSED);
       require(configuration.getActive(), Errors.RESERVE_INACTIVE);
+      require(configuration.getFlashLoanEnabled(), Errors.FLASHLOAN_DISABLED);
```

Comment:
the same require missing in the `validateFlashloanSimple` in some lines above. Because of that looks like a good idea to reuse `simple` ie:

```
...
    require(assets.length == amounts.length, Errors.INCONSISTENT_FLASHLOAN_PARAMS);
    for (uint256 i = 0; i < assets.length; i++) {
      validateFlashloanSimple(reservesData[assets[i]]
        .configuration);
    }

Reply:
Good catch. Let's add tests and evaluate this change!